app/monitoringapi_internal_test.go

```diff
@@ -129,3 +152,11 @@ func TestStartChecker(t *testing.T) {
 		})
 	}
 }
+
+func advanceClock(clock clockwork.FakeClock, duration time.Duration) {
+	// We wrap the Advance() calls with blockers to make sure that the ticker
+	// can go to sleep and produce ticks without time passing in parallel.
+	clock.BlockUntil(1)
```

Comment:
note you added another ticker, so there are more things blocking than previously